xssp_rest/tasks.py

```diff
@@ -94,6 +94,14 @@ def mkhssp_from_sequence(sequence, output_format):
 
     If present, the input FASTA description line is used.
     """
+
+    if output_format == 'hssp_stockholm':
```

Comment:
the format of hg-hssp files is stockholm, so before returning anything the service has to check that this is what the user wants.
Optionally, we could do a conversion if the user wants hssp_hssp instead.